rotkehlchen/chain/substrate/typing.py

```diff
@@ -75,27 +118,42 @@ def chain_explorer_api(self) -> str:
         """
         if self == SubstrateChain.KUSAMA:
             return 'https://kusama.api.subscan.io/api'
+
+        elif self == SubstrateChain.POLKADOT:
+            return 'https://polkadot.api.subscan.io/api'
+
         raise AssertionError(f'Unexpected Chain: {self}')
 
     def substrate_interface_attributes(self) -> SubstrateInterfaceAttributes:
         """Return the attributes for instantiating SubstrateInterface.
         """
         if self == SubstrateChain.KUSAMA:
             return SubstrateInterfaceAttributes(type_registry_preset='kusama')
+
+        elif self == SubstrateChain.POLKADOT:
+            return SubstrateInterfaceAttributes(type_registry_preset='polkadot')
+
         raise AssertionError(f'Unexpected Chain: {self}')
 
     def blocks_threshold(self) -> BlockNumber:
         """Return the blocks difference that marks a node as unsynced.
         """
         if self == SubstrateChain.KUSAMA:
             return BlockNumber(10)
+
+        elif self == SubstrateChain.POLKADOT:
+            return BlockNumber(10)
```

Comment:
Is the same number in both chains?

Reply:
Both Polkadot and Kusama have a 6 seconds block time, so I'd suggest sticking with the default that was set to Kusama.

Reply:
Then I believe you can do something like 
```suggestion
 if self in (SubstrateChain.KUSAMA, SubstrateChain.POLKADOT):
            return BlockNumber(10)
```

Reply:
that's more elegant :) thanks

Reply:
fixed in https://github.com/rotki/rotki/pull/3197/commits/18bed4103906e5e7c81298f91c8abf3a3f242088 thanks!